sdk/src/main/scala/io/horizen/account/api/rpc/service/EthService.scala

```diff
@@ -251,7 +252,10 @@ class EthService(
     applyOnAccountView { nodeView =>
       val secret = getFittingSecret(nodeView.vault, nodeView.state, sender, BigInteger.ZERO)
       val signature = secret.sign(messageToSign)
-      signature.getR ++ signature.getS ++ signature.getV
+      // TODO check order: why V is the last contribution?
```

Comment:
I think it is because of historical reasons first of all. In geth it is the same.
Moreover, I think we have an issue here. Because R and S must be padded to 32 bytes to make the result predictable, otherwise how to understand how to parse?

In geth:
```
// Serialize the signature
	rbytes, sbytes := sigdata.Signature.R.Bytes(), sigdata.Signature.S.Bytes()
	sig := make([]byte, 65)
	copy(sig[32-len(rbytes):32], rbytes)
	copy(sig[64-len(sbytes):64], sbytes)

	// Recover the V value.
	sig, err = makeRecoverableSignature(hash, sig, sigdata.PublicKey)
	if err != nil {
		return nil, err
	}
```